packages/block-editor/src/components/writing-flow/index.js

```diff
@@ -1,7 +1,7 @@
 /**
  * External dependencies
  */
-import { overEvery, find, findLast, reverse, first, last } from 'lodash';
+import { overEvery, find, findLast, reverse, first, last, debounce } from 'lodash';
```

Comment:
Is there a specific reason the logic needs to occur in this component? It's already quite large, and is primarily focused around interactions _between_ blocks.

To me, the proposed behavior reminds me a fair bit of the [`<MultiSelectScrollIntoView>` component](https://github.com/WordPress/gutenberg/tree/master/packages/block-editor/src/components/multi-select-scroll-into-view), which can be used (or not used) independently.

Reply:
Yes, that would be good!

Reply:
Done